packages/core/src/types/config/index.ts

```diff
@@ -94,12 +94,21 @@ export type StorageConfig = (
   FileOrImage;
 
 export type KeystoneConfig<TypeInfo extends BaseKeystoneTypeInfo = BaseKeystoneTypeInfo> = {
-  lists: ListSchemaConfig;
   db: DatabaseConfig<TypeInfo>;
+  graphql?: GraphQLConfig;
+  lists: ListSchemaConfig;
   ui?: AdminUIConfig<TypeInfo>;
   server?: ServerConfig<TypeInfo>;
   session?: SessionStrategy<any>;
-  graphql?: GraphQLConfig;
+  types?: {
+    // TODO: rename to contextTypesPath?
+    // TODO: rename to schemaTypesPath?
+    // TODO: rename to typeInfoPath?
+    // TODO: rename to internalTypesPath?
+    listTypesPath?: string;
```

Comment:
Hard to say, I’ve seen code examples, such as:
- `import { Lists } from '.keystone/types';`
- `import type { Context } from '.keystone/types';`
- `import { TypeInfo } from '.keystone/types';`

I’m not sure what the best name would be.  If everything is in that file, then maybe a more generalized name would be good:
- `internalTypesPath`
- `generatedTypesPath`
- `keystoneTypesPath`

Reply:
@jlarmstrongiv we went with `path`, because we don't have any other examples